Dockerfile

```
@@ -8,10 +8,12 @@ RUN uv sync
 
 # Copy application code
 COPY main.py .
+COPY admin.py .
+COPY templates/ templates/
 COPY assets/ assets/
 
 # Expose the port the app runs on
-EXPOSE 8000
+EXPOSE ${PORT:-8000}
```

Hadolint (2.14.0)

[error] unexpected '}'
expecting '/', a new line followed by the next instruction, or digit

(DL1000)